test/tests/operations/Image.js

```diff
@@ -0,0 +1,103 @@
+        expectedOutput: [
+            "Found 28 tags.",
+            "",
+            "Make: SONY",
+            "Model: DSC-H5",
+            "XResolution: 72",
+            "YResolution: 72",
+            "ResolutionUnit: 2",
+            "Software: Pictomio 1.2.31.0",
+            "ModifyDate: 2010:07:04 23:31:13",
+            "ExposureTime: 0.008",
+            "FNumber: 3.7",
+            "ExposureProgram: 3",
+            "ISO: 200",
+            "DateTimeOriginal: 1220275486",
+            "CreateDate: 1220275486",
+            "ShutterSpeedValue: 6.965784",
+            "ApertureValue: 3.775051",
+            "ExposureCompensation: 0.3",
+            "MaxApertureValue: 3",
+            "MeteringMode: 5",
+            "LightSource: 10",
+            "Flash: 16",
+            "FocalLength: 72",
+            "CustomRendered: 0",
+            "ExposureMode: 1",
+            "WhiteBalance: 1",
+            "SceneCaptureType: 0",
+            "Contrast: 0",
+            "Saturation: 0",
+            "Sharpness: 0",
+        ].join("\n"),
+        recipeConfig: [
+            {
+                op: "From Hex",
+                args: ["Space"]
```

Comment:
Technically this should be "None". It'll work either way at the moment, but if we ever make the 'From Hex' operation more strict, this test will break.